Sources/QuickAction/OAQuickActionRegistry.m

```diff
@@ -314,7 +308,7 @@ -(OAQuickAction *) generateUniqueName:(OAQuickAction *) action
     [self filterQuickActions:TYPE_ADD_ITEMS result:result];
     [self filterQuickActions:TYPE_CONFIGURE_MAP result:result];
     [self filterQuickActions:TYPE_NAVIGATION result:result];
-    [self filterQuickActions:TYPE_CONFIGURE_SCREEN result:result];
+//    [self filterQuickActions:TYPE_CONFIGURE_SCREEN result:result];
```

Comment:
unused

Reply:
TYPE_CONFIGURE_SCREEN = [[OAQuickActionType alloc] initWithIdentifier:0 stringId:@"" class:nil name:OALocalizedString(@"layer_map_appearance") category:CONFIGURE_SCREEN iconName:nil]; Comment out. it's not being used